src/telegrafs/actions/thunks.ts

```diff
@@ -149,8 +164,29 @@ export const addTelegrafLabelsAsync = (
   labels: Label[]
 ): AppThunk<Promise<void>> => async (dispatch): Promise<void> => {
   try {
-    await client.telegrafConfigs.addLabels(telegrafID, labels as ILabel[])
-    const telegraf = await client.telegrafConfigs.get(telegrafID)
+    if (labels.length === 0) {
```

Comment:
Super weird that this is accepting an array of labels. I don't think that's possible from the UI. I would check to see where this being used so that we can:

1. Identify whether we're only ever passing in 1 label or an array

If we find that we are passing in an array, we're going to want to add all the labels, not just the first one.